src/sentry/static/sentry/app/components/organizations/timeRangeSelector/index.jsx

```diff
@@ -329,25 +344,34 @@ class TimeRangeSelector extends React.PureComponent {
                 isAbsoluteSelected={isAbsoluteSelected}
               >
                 <SelectorList isAbsoluteSelected={isAbsoluteSelected}>
-                  {shouldShowRelative && (
-                    <RelativeSelector
-                      onClick={this.handleSelectRelative}
-                      selected={relative || DEFAULT_STATS_PERIOD}
-                    />
-                  )}
-                  {shouldShowAbsolute && (
-                    <SelectorItem
-                      onClick={this.handleAbsoluteClick}
-                      value="absolute"
-                      label={t('Absolute Date')}
-                      selected={isAbsoluteSelected}
-                      last={true}
-                    />
-                  )}
+                  <SelectorListHook
+                    isAbsoluteSelected={isAbsoluteSelected}
+                    relativeSelected={relativeSelected}
+                    handleSelectRelative={this.handleSelectRelative}
+                    handleAbsoluteClick={this.handleAbsoluteClick}
+                    shouldShowRelative={shouldShowRelative}
+                    shouldShowAbsolute={shouldShowAbsolute}
```

Comment:
🚲 🏠 

but when i have a lot of props like this I sometimes write this as:

```jsx
<SelectorListHook
  {...{isAbsoluteSelected, relativeSelected, shouldShowRelative, shouldShowAbsolute}}
  handleAbsoluteClick={this.handleAbsoluteClick}
  handleSelectRelative={this.handleSelectRelative}
/>
```

Feel free to ignore, super subjective opinion.